tdd-seed/src/main/java/org/example/stringCalculator/CalculatorService.java

```diff
@@ -0,0 +1,48 @@
+package org.example.stringCalculator;
+
+import java.io.IOException;
+import java.util.Optional;
+import java.util.Scanner;
+
+public class CalculatorService {
+    private Scanner scanner; // 스캐너로 입력 받은 값
+    private CalculatorMethod calculate;
+
+    public CalculatorService(Scanner scanner) {
+        this.scanner = scanner;
+    }
+
+    public void run() throws IllegalArgumentException {
+        String arithmeticString;
+        boolean flag = true;
+        System.out.println("===============문자열 계산기===============");
+        System.out.println();
+
+        do {
+            System.out.println("문자열 연산을 하기위한 값을 / x + y / 형식으로 입력해주세요. * 종료 = 0 * ");
+            System.out.print("입력 : ");
+            arithmeticString = scanner.nextLine();
+            String target = Optional.ofNullable(arithmeticString).orElse("");
+            if (target.isBlank()) {
+                throw new IllegalArgumentException();
+            }
+            int result = calculateString(arithmeticString);
+            if (result == -1) System.out.println("올바른 형식으로 입력해주세요.");
+            if (result != -1) System.out.println("결과 값은 : " + result);
+            if (result == 0) flag = false;
+        } while (flag);
+    }
+
+    public int calculateString(String str) {
+        if (str.equals("0")) return 0;
+        if (!str.contains(" ")) return -1;
+
+        calculate = new CalculatorMethod();
+        String[] value = calculate.ofSplit(str); // 1 + 2면 value[0]에 1 value[1] + ...
+        int result = 0;
+        for (int i=0; i < value.length -2; i=i+2) {
+            result = calculate.arithmeticResult(calculate.stringToInt(value[0]),
+                    value[i+1].charAt(0), calculate.stringToInt(value[i+2]));
+        } return result;
```

Comment:
```suggestion
        calculate = new CalculatorMethod();
        return calcualte.calculate(str);
```
비즈니스 로직은, 도메인 객체에 맡기는 것이 좋을 것 같아요
각 피연산자들을 계산하고 합을 내는것이 `CalculatorService`의 역할인지 `Calculator`의 역할인지 고민하면 좋을 것 같네요

Reply:
calculator의 역활로 생각해서 calculatorController로 이전 -> 수정완